experimental/maven-plugin/src/main/java/io/github/slsa-framework/SlsaVerificationMojo.java

```diff
@@ -117,41 +118,36 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                     )
                 );
             } catch(MojoExecutionException e) {
-                getLog().info("Skipping slsa verification for " + artifactStr + ": No slsa file found.");
+                getLog().info("Skipping SLSA verification for " + artifactStr + ": No SLSA provenance found.");
                 continue;
             }
 
-            // Verify slsa file
-            try {
-                // Run slsa verification on the artifact and print the result
-                // It will never fail the build process
-                // This might be prone to command-injections. TODO: Secure against that. 
-                String arguments = "verify-artifact --provenance-path ";
-                arguments += "${project.build.directory}/slsa/" + artifact.getArtifactId() + "-" + artifact.getVersion() + "-jar.intoto.build.slsa ";
-                arguments += " --source-uri ./ ${project.build.directory}/slsa/" + artifact.getArtifactId() + "-" + artifact.getVersion() + ".jar";
-                executeMojo(
-                    plugin(
-                        groupId("org.codehaus.mojo"),
-                        artifactId("exec-maven-plugin"),
-                        version("3.1.0")
-                    ),
-                    goal("exec"),
-                    configuration(
-                        element(name("executable"), verifierPath),
-                        element(name("commandlineArgs"), arguments),
-                        element(name("useMavenLogger"), "true")
-                    ),
-                    executionEnvironment(
-                        project,
-                        mavenSession,
-                        pluginManager
-                    )
-                );
-            } catch(MojoExecutionException e) {
-                // TODO: Properly interpret the output based on the verification plugin.
-                getLog().info("Skipping slsa verification: Fail to run slsa verifier.");
-                return;
-            }
+            // Run slsa verification on the artifact and print the result
+            // It will never fail the build process
+            // TODO(#665): Harden against script injections.
+            String arguments = "verify-artifact --provenance-path ";
+            arguments += "${project.build.directory}/slsa/" + artifact.getArtifactId() + "-" + artifact.getVersion() + "-jar.build.slsa ";
```

Comment:
@AdamKorcz please confirm this change is correct (I removed the intoto part of the name)

Reply:
Did you also remove the `catch(`?

Reply:
I did. Build will fail if verification fails. Something i dont fully understand yet, is where the expected repository comes from. I can add a comment

Reply:
LGTM